src/safeposix/syscalls/net_constants.rs

```diff
@@ -384,5 +384,5 @@ pub const EPOLL_CTL_MOD: i32 = 3;
 //for internal use
 #[derive(Debug, PartialEq, Eq, Clone)]
 pub enum ConnState {
-    NOTCONNECTED, CONNECTED, LISTEN, INPROGRESS
+    NOTCONNECTED, CONNECTED, CONNRDONLY, CONNWRONLY, LISTEN, INPROGRESS
```

Comment:
are we always going to do RDONLY | WRONLY to represent RDWRONLY? Maybe it's worth adding RDWR in?